### DISCUSSION TITLE
TLS examples not ported from 1.x to 2.x

### DISCUSSION THREAD

**Discussion by beyonlo** [miguelgrinberg/microdot#236] [General]
Hello

I would like to know if those TLS examples that exists on the 1.3.4 (https://github.com/miguelgrinberg/microdot/tree/v1.3.4/examples/tls)
were not ported to 2.x just because you didn't have time, or because need much changes to be compatible to the 2.x? There is just one TLS example on the 2.0.5 (https://github.com/miguelgrinberg/microdot/tree/v2.0.5/examples/tls)

I'm particularly interested to use the [echo_async_tls.py](https://github.com/miguelgrinberg/microdot/blob/v1.3.4/examples/tls/echo_async_tls.py) + [index.html](https://github.com/miguelgrinberg/microdot/blob/v1.3.4/examples/tls/index.html) with the purpose to have the Microdot running as WebSocket Secure Server (wss) on the Python and MicroPython using asyncio/uasyncio.

Thank you!

**Comment by miguelgrinberg** [comment]
The additional examples do not contribute much, because the application does not need to change at all, except at the bottom where the server is started. To convert an application to use TLS, just change the server start to look like it does in that one TLS example.

**Reply by beyonlo** [reply]
Understood. Thank you!
